x-pack/plugins/security/public/views/management/edit_role/components/privileges/es/__snapshots__/index_privilege_form.test.tsx.snap

```diff
@@ -1,243 +1,113 @@
-              className="indexPrivilegeForm__grantedFieldsRow"
-              describedByIds={Array []}
-              fullWidth={true}
-              hasEmptyLabelSpace={false}
-              helpText={
-                <FormattedMessage
-                  defaultMessage="If no fields are granted, then users assigned to this role will not be able to see any data for this index."
-                  id="xpack.security.management.editRoles.indexPrivilegeForm.grantedFieldsFormRowHelpText"
-                  values={Object {}}
-                />
-              }
-              label={
-                <FormattedMessage
-                  defaultMessage="Granted fields (optional)"
-                  id="xpack.security.management.editRoles.indexPrivilegeForm.grantedFieldsFormRowLabel"
-                  values={Object {}}
-                />
-              }
-            >
-              <React.Fragment>
-                <EuiComboBox
-                  compressed={false}
-                  data-test-subj="fieldInput0"
-                  fullWidth={false}
-                  isClearable={true}
-                  isDisabled={false}
-                  onChange={[Function]}
-                  onCreateOption={[Function]}
-                  options={Array []}
-                  selectedOptions={Array []}
-                  singleSelection={false}
-                />
-              </React.Fragment>
-            </EuiFormRow>
-          </EuiFlexItem>
-        </EuiFlexGroup>
-        <EuiSpacer
-          size="l"
-        />
-        <EuiFlexGroup
-          alignItems="stretch"
-          component="div"
-          direction="column"
-          gutterSize="l"
-          justifyContent="flexStart"
-          responsive={true}
-          wrap={false}
-        >
-          <EuiFlexItem
-            component="div"
-            grow={true}
-          >
-            <EuiSwitch
-              compressed={true}
-              data-test-subj="restrictDocumentsQuery0"
-              label={
-                <FormattedMessage
-                  defaultMessage="Grant read privileges to specific documents"
-                  id="xpack.security.management.editRoles.indexPrivilegeForm.grantReadPrivilegesLabel"
-                  values={Object {}}
-                />
-              }
-              onChange={[Function]}
-              value={false}
-            />
-          </EuiFlexItem>
-        </EuiFlexGroup>
-      </React.Fragment>
-    </EuiFlexItem>
-    <EuiFlexItem
-      component="div"
-      grow={false}
-    >
-      <EuiFormRow
-        describedByIds={Array []}
-        fullWidth={false}
-        hasEmptyLabelSpace={true}
-      >
-        <EuiButtonIcon
-          aria-label="Delete index privilege"
-          color="danger"
-          iconSize="m"
-          iconType="trash"
-          onClick={[MockFunction]}
-          type="button"
-        />
-      </EuiFormRow>
-    </EuiFlexItem>
-  </EuiFlexGroup>
-</React.Fragment>
+<IndexPrivilegeFormUI
```

Comment:
This test isn't testing the actual form anymore, just the intl wrapper. Either need to make this test do a full render (instead of a shallow one) or to test a shallow render of the `IndexPrivilegeFormUI` component rather than the Intl wrapper.